seacrowd/sea_datasets/culturax/culturax.py

```diff
@@ -0,0 +1,146 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+from urllib.parse import urljoin
+
+import datasets
+from pyarrow import parquet as pq
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks
+
+_CITATION = """\
+@article{nguyen2023culturax,
+    author    = {Thuat Nguyen and Chien Van Nguyen and Viet Dac Lai and Hieu Man and Nghia Trung Ngo and Franck Dernoncourt and Ryan A. Rossi and Thien Huu Nguyen},
+    title     = {CulturaX: A Cleaned, Enormous, and Multilingual Dataset for Large Language Models in 167 Languages},
+    journal   = {arXiv preprint arXiv:2309.09400},
+    year      = {2023},
+    url       = {https://arxiv.org/abs/2309.09400},
+}
+"""
+
+_DATASETNAME = "culturax"
+_DESCRIPTION = """\
+CulturaX is a comprehensive multilingual dataset comprising 6.3 trillion tokens across 167
+languages, designed for large language model development. It incorporates an advanced
+cleaning and deduplication process, including language identification and fuzzy
+deduplication with MinHash, to ensure high-quality data for model training. The dataset,
+which spans 16TB in parquet format and 27TB when unpacked, is a combination of the latest
+mC4 and OSCAR corpora, emphasizing non-English languages to support multilingual model
+training. For data cleaning validation, CulturaX employs a SentencePiece tokenizer and
+KenLM language models, utilizing recent Wikipedia dumps for perplexity scoring.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/uonlp/CulturaX"
+_LICENSE = """
```

Comment:
Based on @SamuelCahyawijaya [comments](https://github.com/SEACrowd/seacrowd-datahub/pull/99#discussion_r1404530638) on license, should I wait for that PR #99 to be implemented and add `Licenses.OTHERS.value`?

Reply:
Yeah I think that's the better option. Let's just wait for #99 to be merged and update this.

Reply:
@akhdanfadh @ljvmiranda921, we have just merged https://github.com/SEACrowd/seacrowd-datahub/pull/124. Please update the code accordingly 👍🏻